integration/raft/channel_participation_test.go

```diff
@@ -353,22 +350,152 @@ var _ = Describe("ChannelParticipation", func() {
+			computeSignSubmitConfigUpdate(network, orderer1, peer, c, "participation-trophy")
+
+			currentBlockNumber := nwo.CurrentConfigBlockNumber(network, peer, orderer1, "participation-trophy")
+			Expect(currentBlockNumber).To(BeNumerically(">", 1))
+
+			By("starting third orderer")
+			startOrderer(orderer3)
+			channelparticipation.List(network, orderer3, nil)
+
+			By("joining orderer3 to the channel as a follower")
+			// make sure we can join using a config block from one of the other orderers
+			configBlockPT := nwo.GetConfigBlock(network, peer, orderer2, "participation-trophy")
+			expectedChannelInfoPTFollower := channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "onboarding",
+				ClusterRelation: "follower",
+				Height:          0,
+			}
+			channelparticipation.Join(network, orderer3, "participation-trophy", configBlockPT, expectedChannelInfoPTFollower)
+
+			By("ensuring orderer3 completes onboarding successfully")
+			expectedChannelInfoPTFollower.Status = "active"
+			expectedChannelInfoPTFollower.Height = 4
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfoPTFollower))
+
+			By("adding orderer3 to the consenters set")
+			channelConfig = nwo.GetConfig(network, peer, orderer1, "participation-trophy")
+			c = configtx.New(channelConfig)
+			err = c.Orderer().AddConsenter(consenterChannelConfig(network, orderer3))
+			Expect(err).NotTo(HaveOccurred())
+			computeSignSubmitConfigUpdate(network, orderer1, peer, c, "participation-trophy")
+
+			By("ensuring orderer3 transitions from follower to member")
+			// config update above added a block
+			expectedChannelInfoPT.Height = 5
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfoPT))
+
+			// make sure orderer3 finds the leader
+			newLeader := findLeader([]*ginkgomon.Runner{ordererRunners[2]})
+			Expect(newLeader).To(Equal(leader))
+
+			By("submitting transaction to orderer3 to ensure it is active")
+			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello-again"))
+			resp, err := ordererclient.Broadcast(network, orderer3, env)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
+			expectedBlockNumPerChannel := map[string]int{"participation-trophy": 5}
+			assertBlockReception(expectedBlockNumPerChannel, orderers, peer, network)
+
+			By("checking the channel height")
+			expectedChannelInfoPT.Height = 6
+			channelInfo := channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			Expect(channelInfo).To(Equal(expectedChannelInfoPT))
```

Comment:
Nit: can't we use the submitTxn() helper here?

Reply:
🧹